dss/api/subscriptions.py

```diff
@@ -106,15 +110,14 @@ def put(json_request_body: dict, replica: str):
 
     # Queries are unlikely to fit in all of the indexes, therefore errors will almost always occur. Only return an error
     # if no queries are successfully indexed.
-    if not subscribed_indexes:
+    if len(doc_indexes) > 0 and not subscribed_indexes:
```

Comment:
good catch

Reply:
This is a valid use case - adding subscriptions after initial deployment yet before documents are indexed. This is something we wanted to do before but couldn't - and now can.